package.json

```diff
@@ -29,10 +29,10 @@
     "eslint-plugin-react": "^7.12.4"
   },
   "dependencies": {
+    "babel-eslint": "^10.0.0",
```

Comment:
We're using `babel` as the parser in the [base config](https://github.com/ebates-inc/eslint-config/blob/master/index.js#L12) so it should be a dependency.